MetadataExtractor/Extractors/Data/IDataExtract.py

```diff
@@ -3,4 +3,4 @@
 
 class IDataExtract(IExtract):
     def registerMimeTypes(self):
-        self.mimeTypes["concrete"] = ["text/csv", "text/xml", "application/json"]
+        self.mimeTypes["concrete"] = ["text/csv", "text/xml", "application/json", "dcm"]
```

Comment:
Please remove that from the default, I don't think this is appropriate as a default value (and won't be necessary for your case).